sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog for the Azure Data Factory V2 .NET SDK
 
+## Version 4.17.1
+###  Feature Additions
+- Fixed FactoryIdentity constructor Type has a default value when Type did not assigne value
```

Comment:
"Fixed potential MI type missing issue by using system assigned MI as default " is a better notes.